### PR TITLE
charts: adapting resource requests for tests

### PR DESCRIPTION
**Pull request by ableuler**
After merging #2011 the tests run at pretty exactly at 2GB of memory and <= 1 CPU.

/deploy

### CODE REVIEW

File: helm-chart/renku/templates/tests/test-renku.yaml [hunk 1]
```diff
@@ -80,11 +80,11 @@ spec:
...
-        memory: "4G"
-        cpu: "2"
+        memory: "3G"
+        cpu: "1500m"
         ephemeral-storage: "1G"
       limits:
-        memory: "4G"
-        cpu: "2"
+        memory: "3G"
+        cpu: "1000m"
```
Comment (aledegano) [inline helm-chart/renku/templates/tests/test-renku.yaml:88]: Is it normal that the limit is lower than the request?

Comment (pameladelgado) [reply helm-chart/renku/templates/tests/test-renku.yaml:88]: is normally the other way around, probably was a typo...

**Comment by pameladelgado** [comment]
what about this particular run? https://github.com/SwissDataScienceCenter/renku/runs/2077083776
do you know if it failed because of another non-resource-related reason?

File: helm-chart/renku/templates/tests/test-renku.yaml (hunk 1 above)
Comment (ableuler) [reply helm-chart/renku/templates/tests/test-renku.yaml:88]: oh gosh, I seem to be unable to edit these properly through the gitlab web UI...

**Review by pameladelgado — APPROVED** [review]
LGTM

**Pull request merged**